docs/resources/ui-building-blocks/components/built-in-components/expandable.md

```diff
@@ -0,0 +1,33 @@
+
+
+# Expandable 
```

Comment:
seems like it's WIP

Reply:
Components are not added to this PR completely, I did whatever I was able to do in the time span. But technically its part of another ticket. https://linear.app/flutterflow/issue/DEVR-403/built-in-components